src/foam/nanos/fs/File.js

```diff
@@ -34,7 +34,10 @@ foam.CLASS({
     {
       class: 'Blob',
       name: 'data',
-      documentation: 'File data'
+      documentation: 'File data',
+      adapt: function(oldObj, newObj) {
```

Comment:
This should be documented, otherwise it looks like a NOP and someone is likely to delete it.